modules/rhythmoneBidAdapter.js

```diff
@@ -34,10 +31,7 @@ function RhythmoneAdapter (bidManager, global, loader) {
 
   function load(bidParams, url, callback) {
     loader(url, function(responseText, response) {
-      if (response.status === 200)
-      { callback(200, 'success', response.responseText); }
-      else
-      { callback(-1, 'http error ' + response.status, response.responseText); }
+      if (response.status === 200) { callback(200, 'success', response.responseText); } else { callback(-1, 'http error ' + response.status, response.responseText); }
```

Comment:
I'm not sure why it doesn't do that... but I agree. `max-len` is definitely an eslint rule we can add if we want, though.

For now I'll split it up manually.